md3-themes/MD3BluiDarkTheme.ts

```diff
@@ -48,10 +48,12 @@ export const MD3BluiDarkTheme = {
         onSurfaceDisabled: 'rgba(26, 27, 30, 0.38)',
         backdrop: 'rgba(45, 48, 56, 0.4)',
         // @TODO verify disabled color for dark theme
-        disabled: '#192024',
+        disabled: '#EBF1F4',
```

Comment:
This is the correct color for disabled, light and dark theme

<img width="596" alt="image" src="https://github.com/etn-ccis/blui-react-native-showcase-demo/assets/8997218/3730c0ef-f526-4f68-a2dc-883c3bbc2665">